sdk/core/core/test/az_test.h

```diff
@@ -13,3 +15,5 @@ extern int exit_code;
       exit_code = 1; \
     } \
   } while (false);
+
+#define TEST_EXPECT_SUCCESS(exp) TEST_ASSERT(!az_failed(exp))
```

Comment:
az_succeeded() would be better than !az_failed()

Reply:
Fixed.